### PR TITLE
Partisn dagmc

### PR DESCRIPTION
**Pull request by kkiesling**
Updated the `partisn.py` module to work with DAGMC as an optional dependency. Addresses #17.

### CODE REVIEW

File: pyne/partisn.py [hunk 1]
```diff
@@ -41,9 +41,14 @@
 
 print(HAVE_PYMOAB)
```
Comment (gonuke) [inline pyne/partisn.py]: This isn't from you're PR, but I wonder if there is a reason for including this print statement here?

**Review by gonuke — COMMENTED** [review]
This LGTM - what are your thoughts on adding a test that uses a simple 2x1x1 grid with manually determined `dg` and `mat_assigns`?

File: pyne/partisn.py (hunk 1 above)
Comment (kkiesling) [reply pyne/partisn.py]: I have no idea why that was there. I can remove it. I don't see it as necessary.

**Comment by kkiesling** [comment]
Updated one of the tests that already took in optional arguments to include a manually set `dg` and `mat_assigns` object. If CI passes, should be good to go.

**Comment by gonuke** [comment]
Thanks @kkiesling

**Pull request merged**